io.gitlab.adhami3310.Footage.json

```diff
@@ -0,0 +1,260 @@
+{
+    "id": "io.gitlab.adhami3310.Footage",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "44",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "add-extensions": {
+        "org.pitivi.Pitivi.Codecs": {
+            "directory": "lib/codecs",
+            "add-ld-path": "lib",
+            "bundle": true,
+            "autodelete": true
+        }
+    },
+    "cleanup-commands": [
+        "mkdir -p ${FLATPAK_DEST}/lib/ffmpeg"
+    ],
+    "command": "footage",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--socket=pulseaudio"
+    ],
+    "build-options": {
+        "append-path": "/usr/lib/sdk/rust-stable/bin",
+        "env": {
+            "GST_PLUGIN_PATH": "/app/lib/codecs/lib/gstreamer-1.0",
+            "GST_PLUGIN_SYSTEM_PATH": "/app/lib/gstreamer-1.0/",
+            "GST_ENCODING_TARGET_PATH": "/app/share/gstreamer-1.0/encoding-profiles/:/app/share/pitivi/encoding-profiles/",
+            "GST_VAAPI_ALL_DRIVERS": "1",
+            "CARGO_HOME": "/run/build/footage/cargo"
+        },
+        "append-ld-library-path": "/app/lib/codecs/lib",
+        "ldflags": "-L/app/lib/codecs/lib"
+    },
+    "cleanup": [
+        "/include",
+        "/lib/*.a",
+        "/lib/*.la",
+        "/lib/gstreamer-1.0/include",
+        "/lib/pkgconfig",
+        "/share/man"
+    ],
+    "modules": [
+        {
+            "name": "blueprint-compiler",
+            "buildsystem": "meson",
+            "cleanup": [
+                "*"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.gnome.org/jwestman/blueprint-compiler.git",
+                    "tag": "v0.8.1",
+                    "commit": "aa7679618e864748f4f4d8f15283906e712752fe"
+                }
+            ]
+        },
+        "codecs/liba52.json",
+        "codecs/libmpeg2.json",
```

Comment:
Any reason these two are separate but the rest isn't? I recommend keeping everything in the single manifest.

Reply:
They use patches and such so I followed what Pitivi did.

Reply:
it's not a model.